catalog/view/theme/default/template/extension/payment/razorpay.twig

```diff
@@ -69,12 +69,10 @@
           el.value = 'Please wait...';  
         }
       } else {
-        if(!razorpay_instance){
```

Comment:
@ramth05 Why this condition has been removed?

Reply:
It's using old cached instance